frontend/src/components/RecipeList.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+
+import IndividualRecipe from "./IndividualRecipe"
+
+class RecipeList extends Component{
+
+    render(){
+        return (
+            <div className="recipeListPage">
+                <div className="topBar">
+                    <div className="newRecipeLink"><Link to="/create-recipe">Add Recipe</Link></div>
+                    <input />
+                    <div className="filterSection">
+                        <div className="filterButton"> Filter </div>
+                        <div className="filterContent">
+                            <form>
```

Comment:
When we code we want to keep our code as less repetitive as possible. So in this case instead of just rendering 5 inputs manually. How about you map through an array of terms?

```{['Breakfast', 'Lunch', 'Dinner', 'Dessert', 'Snack'].map(item => <input name="filter" type="radio">{item}</input>)}```